src/mappings/entities/mapExtrinsic.ts

```diff
@@ -23,7 +22,7 @@ export function createExtrinsic(extrinsic: SubstrateExtrinsic): Extrinsic {
     signedbyAddress: signedbyAddress ? 1 : 0,
     address,
     nonce: extrinsic.extrinsic.nonce.toNumber(),
-    extrinsicHash: hexStripPrefix(extrinsic.extrinsic.hash.toJSON()),
+    extrinsicHash: extrinsic.extrinsic.hash.toJSON(),
```

Comment:
IMO either we leave this without the prefix, or mark it as a breaking change.

It seems weird to me if someone didn't resync to have a mix of prefix and non prefix values in their DB.

I do think its better with the prefix though, but if we are adding it in it should be clear the response changed for people.

Reply:
I think this was left by mistake. Even the SDK directly sends 0x prefixed hash value in `getTransactionByHash`, so feels like a fix to me. We can add in migration to update the existing values

Reply:
I think this was because harvester didn't prefix them and we had to run tooling with the exact data

I think an update is better, but a migration so its consistent would be good, and it should  be marked as a feature, not as a chore.